packages/atlas/src/hooks/useTimeMismatchWarning.ts

```diff
@@ -16,7 +16,7 @@ export const useTimeMismatchWarning = () => {
 
   useEffect(() => {
     if (!timeMismatchDisabled) {
-      axios.get('http://worldtimeapi.org/api/ip').then((response) => {
+      axios.get('https://worldtimeapi.org/api/ip').then((response) => {
         const serverTime = response.data.unixtime
         const clientTime = Date.now()
```

Comment:
There will be always a mismatch - API returns epoch in seconds and `Date.now()` in milliseconds